docs/integrations/sources/metabase.md

```diff
@@ -69,8 +69,9 @@ The Metabase source connector supports the following [sync modes](https://docs.a
 
 ## Changelog
 
-| Version | Date       | Pull Request                                                 | Subject                    |
-|:--------|:-----------|:-------------------------------------------------------------|:---------------------------|
-| 0.3.0   | 2022-11-09 | [19236](https://github.com/airbytehq/airbyte/pull/19236)     | Migrate to YAML.           |
-| 0.2.0   | 2022-10-28 | [18607](https://github.com/airbytehq/airbyte/pull/18607)     | Disallow using `http` URLs |
-| 0.1.0   | 2022-06-15 | [6975](https://github.com/airbytehq/airbyte/pull/13752)      | Initial (alpha) release    |
+| Version | Date       | Pull Request                                             | Subject                    |
+|:--------|:-----------|:---------------------------------------------------------|:---------------------------|
+| 0.3.1   | 2022-12-15 | [20530](https://github.com/airbytehq/airbyte/pull/20530) | Run on CDK 0.15.0          |
```

Comment:
nit: PR # is 20535 now

Reply:
Thanks! Will remember for the others now